firefox-ios/Client/Frontend/Home/Homepage Rebuild/TopSites/TopSitesDimensionImplementation.swift

```diff
@@ -17,6 +42,8 @@ struct TopSitesDimensionImplementation {
             availableWidth = availableWidth - cellWidth - HomepageSectionLayoutProvider.UX.standardSpacing
         }
         let minCardsConstant = HomepageSectionLayoutProvider.UX.TopSitesConstants.minCards
-        return numberOfTiles < minCardsConstant ? minCardsConstant : numberOfTiles
+        let numberOfTilesPerRow = numberOfTiles < minCardsConstant ? minCardsConstant : numberOfTiles
```

Comment:
Originally, I had it setting the class property, but since there's a follow up ticket for that call. I wanted to keep return the value to be used for determining the layout separate from the dispatch call. Any issues if I make this update after I do further investigation on that ticket?